great_expectations/profile/base.py

```diff
@@ -18,11 +18,87 @@
 logger = logging.getLogger(__name__)
 
 
+class OrderedEnum(Enum):
+    def __ge__(self, other):
+        if self.__class__ is other.__class__:
+            return self.value >= other.value
+        return NotImplemented
+
+    def __gt__(self, other):
+        if self.__class__ is other.__class__:
+            return self.value > other.value
+        return NotImplemented
+
+    def __le__(self, other):
+        if self.__class__ is other.__class__:
+            return self.value <= other.value
+        return NotImplemented
+
+    def __lt__(self, other):
+        if self.__class__ is other.__class__:
+            return self.value < other.value
+        return NotImplemented
+
+
+class OrderedProfilerCardinality(OrderedEnum):
+    NONE = 0
+    ONE = 1
+    TWO = 2
+    VERY_FEW = 3
+    FEW = 4
+    MANY = 5
+    VERY_MANY = 6
+    UNIQUE = 7
+
+    @staticmethod
+    def get_basic_column_cardinality(num_unique=None, pct_unique=None):
+        if num_unique is None or num_unique == 0 or pct_unique is None:
+            cardinality = "NONE"
+        elif pct_unique == 1.0:
+            cardinality = "UNIQUE"
+        elif num_unique == 1:
+            cardinality = "ONE"
+        elif num_unique == 2:
+            cardinality = "TWO"
+        elif num_unique < 20:
+            cardinality = "VERY_FEW"
+        elif num_unique < 60:
+            cardinality = "FEW"
+        elif pct_unique > 0.1:
+            cardinality = "VERY_MANY"
+        else:
+            cardinality = "MANY"
+
+        return cardinality
+
+    # def __init__(self, config=None, config_parser=None):
```

Comment:
Should this commented out section be removed? Or is it here for future reference? If for reference I would suggest adding a TODO note above it clarifying.